python/src/TensorizedUniVariateFunctionFactory_doc.i.in

```diff
@@ -0,0 +1,36 @@
+%feature("docstring") OT::TensorizedUniVariateFunctionFactory
+"Base class for tensorized multivariate functions.
+
+Available constructors:
+        TensorizedUniVariateFunctionFactory(*functions*)
+
+        TensorizedUniVariateFunctionFactory(*functions, enumerateFunction*)
+
+Parameters
+----------
+functions : list of :class:`~openturns.UniVariateFunctionFamily`
+    List of univariate function factories.
+enumerateFunction : :class:`~openturns.EnumerateFunction`
+    Associates to an integer its multi-index image in the :math:`\Nset^d`
+    dimension, which is the dimension of the basis. This multi-index represents
+    the collection of degrees of the univariate polynomials.
+
+Notes
+-----
+TensorizedUniVariateFunctionFactory is a particular case of implementation of
+the :class:`~openturns.Basis` in the case of linear regression. 
```

Comment:
No link with linear regression here! It simply allows to create multidimensional functions as the tensor product of univariate functions created by their respective factories. These functions can be of any kind, polynomial or not, orthgonal wrt a specific measure or not.